app.rb

```diff
@@ -0,0 +1,143 @@
+require './book'
+require './student'
+require './teacher'
+require './person'
+require './rental'
```

Comment:
- [ ] Kindly replace all `require` keywords with the word `require_relative`. When I try to run the app, I get the following errors and that is because of the `require` keyword used:

![image](https://user-images.githubusercontent.com/77106962/168378636-e7b28640-8f2e-463e-b831-a53066d9c157.png)

Kindly change it. 👍